### PR TITLE
client/eth: Configure gas fee limit

### PR DESCRIPTION
**Pull request by martonp**
Allow the gas fee limit for the wallet to be configured.

### CODE REVIEW

**Comment by chappjc** [comment]
New conflict after https://github.com/decred/dcrdex/pull/1335/files was merged

**Review by chappjc — APPROVED** [review]

**Review by JoeGruffins — APPROVED** [review]
Love the concise pr.

File: client/asset/eth/eth.go [hunk 1]
```diff
@@ -90,6 +90,25 @@ var (
...
+	err = config.Unmapify(settings, &cfg)
+	if err != nil {
+		return nil, fmt.Errorf("error parsing wallet config: %w", err)
+	}
+	return cfg, nil
+}
+
+// Check that Driver implements asset.Driver.
+var _ asset.Driver = (*Driver)(nil)
+var _ asset.Creator = (*Driver)(nil)
```
Comment (chappjc) [inline client/asset/eth/eth.go:110]: Oh, sorry for another iteration, but the resolution was to remove these duplicate lines because they are also below.

Comment (martonp) [reply client/asset/eth/eth.go]: My bad, I fixed it now.

**Comment by martonp** [comment]
@chappjc Thanks for rebasing, I was still running the harness tests, they're all good though.

**Comment by chappjc** [comment]
Ooops sorry about that.  I wanted to get that so you could get to #1325 .

**Pull request merged**